python/ray/data/tests/test_dataset.py

```diff
@@ -2961,6 +2971,24 @@ def test_groupby_arrow(ray_start_regular_shared):
             "value")
 
 
+def test_groupby_arrow_std(ray_start_regular_shared):
```

Comment:
Can we also test std for simple dataset as well?

Reply:
Ah good catch!